ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.15__amd64-stable-4.15-upgrade-from-stable-4.13.yaml

```diff
@@ -0,0 +1,95 @@
+base_images:
+  ansible:
+    name: "4.15"
+    namespace: ocp
+    tag: ansible
+  cli:
+    name: "4.15"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  hypershift-operator:
+    name: "4.15"
+    namespace: ocp
+    tag: hypershift-operator
+  openstack-installer:
+    name: "4.15"
+    namespace: ocp
+    tag: openstack-installer
+  tests-private-postupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.15"
+  tests-private-preupg:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.15"
+    namespace: ocp
+    tag: tools
+  verification-tests:
+    name: verification-tests
+    namespace: ci
+    tag: latest
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  intermediate414:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.14"
+  latest:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.13"
+  target:
+    release:
+      architecture: amd64
+      channel: candidate
+      version: "4.15"
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 200Mi
+tests:
+- as: gcp-ipi-baselinecaps-none-f14
+  cron: 12 22 13,27 * *
+  steps:
+    cluster_profile: gcp-qe
+    env:
+      BASELINE_CAPABILITY_SET: None
+      PERSISTENT_MONITORING: "false"
+      UPGRADE_CHECK_RUN_TAGS: '@upgrade-check and @4.14 and @gcp-ipi and not @fips'
+      UPGRADE_PRE_RUN_TAGS: '@upgrade-prepare and @4.14 and @gcp-ipi and not @fips'
```

Comment:
For `baselinecaps-none` profiles, maybe we need to exclude test cases from storage/console/workloads 
@duanwei33 @yanpzhan @kasturinarra could you double confirm ?

Reply:
Thanks for the reminder, yes please exclude storage cases for cucushift: not @storage

Reply:
@liangxia yes, workload cases can be disabled, also i think we should not run any other cases from other teams because from 4.15 there is a new feature that was introduced where deployer and builder sa wont be created if baselineCapabilitySet is None and due to this when new-project gets created, tests might fail due to that. 
[1] https://issues.redhat.com/browse/WRKLDS-916

Reply:
hi @kasturinarra can I use **not @workloads and not @console** to exclude workloads and console related cases?

Reply:
yes, you can use that